aws-concepts/index.md

```diff
@@ -6,6 +6,10 @@ thumbnail: /assets/glossary/aws-concepts-glossary.png
 sidebar: false
 prev: false
 next: false
+
+datePublished: "2024-01-15"
```

Comment:
Let's update the index pages always when something happened even in the subpages